Modules/Sources/WordPressUI/Extensions/UIView+AutoLayout.swift

```diff
@@ -0,0 +1,100 @@
+import UIKit
+import SwiftUI
+
+extension UIView {
+    /// Pins edges of the view to the edges of the given container. By default,
+    /// pins to the superview.
+    @discardableResult
+    public func pinEdges(
```

Comment:
It won't be clear what it does with a default argument `view.pin(to: view)`. It has to have the default argument because you'll use `.all` 99% of the time.